alica_tests/src/test/etc/nase/Alica.yaml

```diff
@@ -62,6 +62,9 @@ Alica:
     Enabled: true
     LogFolder: /tmp/log
 
+  Logging:
+    Verbosity: DEBUG
```

Comment:
use `INFO`, debug spams at the moment

Reply:
changed the level to INFO in https://github.com/rapyuta-robotics/alica/pull/302